develop/plone/sessions/cookies.rst

```diff
@@ -105,8 +114,7 @@ Language cookie
 ---------------
 
 ``I18N_LANGUAGE`` is set by ``portal_languages`` tool.
-Disable it by *Use cookie for manual override* setting in
-``portal_languages``.
+Disable it by *Use cookie for manual override* setting in ``portal_languages``.
```

Comment:
Is that a GUI thing? If so, we have a directive for that:

```
by :guilabel:`Use cookie for manual override`
```

Also how does one disable it? Tick/untick a checkbox?

Also what does disabling the setting actually do and why would someone do it? This paragraph lacks an explanation for why to do this.

Reply:
This one was missed.